### ISSUE TITLE
Feature request - HiDiffusion 

### ISSUE THREAD

**Issue by atxoxx** [yolain/ComfyUI-Easy-Use#147]
Hi, could you add support for Hidiffusion ? 
It's a real game changer for sd1.5 and sdxl image generation 
basically it allows to make "native" sd1.5 image generation in 1024*1024,2048*2048
and for sdxl 2048*2048 and 4096*4096 without any issue (multiple people, strange limbs etc) 

here's the github with the code : https://github.com/megvii-research/HiDiffusion

**Comment by yolain** [comment]
It's too hard for me.😂but i see the author is trying to implement in comfyui.

**Comment by atxoxx** [comment]
maybe I can help ? :D

**Comment by whmc76** [comment]
i have submit a request to the comfy hidiffusion node, maybe they will make a easier apply method

**Comment by yolain** [comment]
> i have submit a request to the comfy hidiffusion node, maybe they will make a easier apply method

There are two custom nodes package so far, I did try it a couple of days ago but it didn't work very well.

https://github.com/blepping/comfyui_jankhidiffusion
https://github.com/florestefano1975/ComfyUI-HiDiffusion

**Comment by atxoxx** [comment]
also this one : https://github.com/kijai/ComfyUI-ELLA-wrapper/commit/457c9b8a1ea5d98ba9c840481a7bf5552fdc7f20

maybe worth a look

**Comment by yolain** [comment]
> also this one : [kijai/ComfyUI-ELLA-wrapper@457c9b8](https://github.com/kijai/ComfyUI-ELLA-wrapper/commit/457c9b8a1ea5d98ba9c840481a7bf5552fdc7f20)
> 
> maybe worth a look I mostly use this one and it work perfectly

Yeah, I know this one too. But they all need to go through diffusers, not really native comfy.

**Comment by atxoxx** [comment]
So I guess best solution is to wait hidiffusion team to make something easier to implement
